tools/common/pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <groupId>jakarta.tck</groupId>
     <artifactId>common</artifactId>
-    <version>11.0.0-SNAPSHOT</version>
+    <version>11.0.0-RC7</version>
```

Comment:
Is this part right? I wouldn't think we'd want this pushed to main because every time it's built locally it will replace what has, potentially, been downloaded from Maven.

Reply:
Correct, that is not right. Fixed.